jenkins-scripts/lib/dependencies_archive.sh

```diff
@@ -659,6 +659,7 @@ IGN_PHYSICS_DEPENDENCIES="libbenchmark-dev \\
                           libignition-math6-dev \\
                           libignition-math6-eigen3-dev \\
                           libignition-plugin-dev \\
+                          libsdformat9-dev \\
```

Comment:
looks weird to me to have two versions of sdformat as dependencies listed together. I assume we are abusing the fact that we can co-install them to solve problems with different versions.

Reply:
Yeah I'd lean towards picking the SDF version based on `IGN_PHYSICS_MAJOR_VERSION`, but I approved because I thought there was an urgency.

Reply:
> we are abusing the fact that we can co-install them to solve problems with different versions

this is correct; I just did it this way because it was more concise

if you like @j-rivero we can fix it in this PR or in a subsequent one

Reply:
no big deal anyway.

Reply:
I've made the proper adjustment in #196